apps/wallet/src/background/Window.ts

```diff
@@ -4,7 +4,8 @@
 import { filter, fromEventPattern, share, take, takeWhile } from 'rxjs';
 import Browser from 'webextension-polyfill';
 
-import { POPUP_HEIGHT, POPUP_WIDTH } from '_src/ui/app/wallet/constants';
+const POPUP_WIDTH = 360;
+const POPUP_HEIGHT = 600;
```

Comment:
why remove this `import { POPUP_HEIGHT, POPUP_WIDTH } from '_src/ui/app/wallet/constants';`

Reply:
it was only used here and I wanted to delete the wallet/constants file